api/docs/bt.dox

```diff
@@ -327,6 +327,19 @@ passes an \c instrlist_t of all Level 3 \c instr_t's, for simplicity.
   <tr><td><table width=100% bgcolor="#0000C0" cellspacing=0 cellpadding=1 border=0>
   <tr><td></td></tr></table></td></tr></table></td></tr></table>
 \endhtmlonly
+
+\if level_of_detail
```

Comment:
Remove the \if: that define strips out IR levels which we no longer expose to users.  It's unrelated to the new docs here.

Reply:
Done